src/Sfx/App/Scripts/types/cytoscape.ts

```diff
@@ -0,0 +1,2 @@
+
+declare var cytoscape:any;
```

Comment:
is there not a types declaration for this for typescript support? There should be a library @types/<something > for adding these.